downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md

```diff
@@ -167,7 +167,7 @@ type Employee record {|
 
 ### Improvements related to JSON compatibility
 
-Three langlib functions were introduced and `toJsonString` is now allowed to have `anydata`.
+Three new langlib methods have been introduced to improve converting to and from JSON. Additionally, the `toJsonString` method can now be called on `anydata` values.
 
 #### `toJson`
```

Comment:
Also, should we consider making these headings a bit more descriptive? When viewing the generated page, this looks weird with the formatting.

Reply:
We have a bit more content now. :smile:

Reply:
No no, I meant the heading `toJson` itself.

Reply:
If we are changing this, the only change I can think of is making it something like
```suggestion
#### The `toJson` method
```

Reply:
Yeah +1. That's what I was thinking about as well